packages/rrweb/src/record/mutation.ts

```diff
@@ -533,10 +533,18 @@ export default class MutationBuffer {
       this.attributes.push(item);
       this.attributeMap.set(textarea, item);
     }
-    item.attributes.value = Array.from(
+    let value = Array.from(
```

Comment:
You're getting an eslint error here. Should be the following to fix it:
```suggestion
    const value = Array.from(
```